cartography/intel/aws/ec2/instances.py

```diff
@@ -308,11 +308,6 @@ def cleanup(neo4j_session: neo4j.Session, common_job_parameters: Dict[str, Any])
     logger.debug("Running EC2 instance cleanup")
     GraphJob.from_node_schema(EC2ReservationSchema(), common_job_parameters).run(neo4j_session)
     GraphJob.from_node_schema(EC2InstanceSchema(), common_job_parameters).run(neo4j_session)
-    GraphJob.from_node_schema(EC2SubnetSchema(), common_job_parameters).run(neo4j_session)
```

Comment:
These are all moved from here to their respective modules. Specifically,

- EC2SubnetSchema -> intel.aws.ec2.subnets.cleanup_subnets
- EC2SecurityGroupSchema ->  intel.aws.ec2.security_groups.cleanup_ec2_security_groupinfo
- EC2KeyPairSchema -> intel.aws.ec2.key_pairs.cleanup_ec2_key_pairs
- EC2NetworkInterfaceSchema -> intel.aws.ec2.network_interfaces.cleanup_network_interfaces
- EBSVolumeSchema -> intel.aws.ec2.cleanup_volumes